spyder/plugins/ipythonconsole/utils/messagehandler.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+import logging
+logger = logging.getLogger(__name__)
+
+class SpyderMessageHandler:
+    """Message handler for spyder messages. Each shell widget has one instance of it to
+handle the messages sent by it's kernel. Using the IPythonAPIMixin plugins can register additional global
```

Comment:
*its , *IPythonAPIMixin, (I was rather confused on the meaning without the missing comma)